docs/changes.md

```diff
@@ -0,0 +1 @@
+update the db2 role documenation to add a link to the db2 documentation for reference for the storage classes.
```

Comment:
You don't need make changes / add this new file.

Reply:
I deleted the file